_posts/2020-03-30-team-working-agreements.md

```diff
@@ -0,0 +1,314 @@
+team.
+
+### It takes effort to make sure one person isn't introducing their bias into the working agreement.
+
+It's important for the team agreement brainstorm to provide an environment in which every team member is heard.
+Even if your team is successful at this, there are still opportunities for individual bias to affect the working
+agreement.
+
+We saw earlier how bias can appear when the examples provided for the brainstorm are too narrowly scoped.
+
+We recognized another bias vector in the distillation of the brainstorm into an actual agreement. If this is
+handled by a single person, it's very possible for them to produce a working agreement that misrepresents _the
+rest_ of the team. To counter this, we've had multiple team members pair on the distillation process. Feedback on
+the initial draft of the agreement is also important to ensure it does not misrepresent the team's ideas.
+
+The problem of introducing bias to the distillation process is most challenging with topics that are disputed
+across the team. One of our teams spent a lot of time talking about meetings in their brainstorm, but struggled to
+come to consensus on them. No single person, or even a pair, could add a disputed topic like this to the working
+agreement without introducing their personal bias. In this specific case, the team chose to leave meetings off the
+working agreement until they could reach consensus. Topics that are disputed across the team are important to talk
+about at _some_ point, but the brainstorm is probably not the time.
+
+### Team health is easy to postpone when you have important product work to do.
+
+One of our teams put the act of distilling the brainstorm notes into an agreement on one person. That person became
+busier than expected, didn't have time to write the agreement, and the team's agreement stalled.
+
+Just as personal self-care gets put on the back burner during stressful times, creating a working agreement is easy
+to bump down the backlog during intense periods of sprint work. The longer a team drags out the working agreement
+process, the more context is lost from the original brainstorm. We recommend not letting your working agreement
+linger.
+
+It's also important to revisit the working agreement often. It is a living document, not one-and-done. Review the
+agreement as a team occasionally; update the agreement as your team learns how to collaborate better; cultivate the
+agreement. If an agreement gets stale, it will become inaccurate, meaningless, and unused.
+
+### What does it even mean to have a team working agreement?
+
+A team working agreement is a neat artifact. The brainstorm process has great value. But there is non-obvious work
+involved in turning a team working agreement into the team culture. We have many questions that we aren't sure how
+to answer:
+
+- How do you enforce the agreement?
+- **Should** you enforce the agreement?
```

Comment:
Again minor, but it feels weird to only have one item bolded. Maybe italics makes more sense?

![Screen Shot 2020-03-30 at 14 31 07](https://user-images.githubusercontent.com/498212/77948238-1a95b800-7293-11ea-867e-620f6911099e.png)


```suggestion
- _Should_ you enforce the agreement?
```